packages/langchain/src/openai/README.md

```diff
@@ -46,30 +44,48 @@ import {
 // For a chat client
 const chatClient = new AzureOpenAiChatClient({ modelName: 'gpt-4o' });
 // For an embedding client
-const embeddingClient = new AzureOpenAiEmbeddingClient({ modelName: 'gpt-4o' });
+const embeddingClient = new AzureOpenAiEmbeddingClient({
+  modelName: 'text-embedding-3-small'
+});
 ```
 
 In addition to the default parameters of Azure OpenAI and LangChain, additional parameters can be used to help narrow down the search for the desired model:
 
 ```ts
 const chatClient = new AzureOpenAiChatClient({
   modelName: 'gpt-4o',
-  modelVersion: '24-07-2021',
+  modelVersion: '2024-08-06',
   resourceGroup: 'my-resource-group'
 });
 ```
 
-**Do not pass a `deployment ID` to initialize the client.**
-For LangChain model clients, initialization requires specifying the model name, model version, and resource group.
```

Comment:
This was confusing to me that model version and resource group are optional. But then said again with the word `required`.